include/hc_am.hpp

```diff
@@ -19,7 +19,7 @@ struct AmPointerInfo {
     void *      _hostPointer;   ///< Host pointer.  If host access is not allowed, NULL.
     void *      _devicePointer; ///< Device pointer.  
     size_t      _sizeBytes;     ///< Size of allocation.
-    hc::accelerator &_acc;       ///< Device / Accelerator to use.
```

Comment:
@facao and @bensander . after our discussion I think it's better to change this field to hc::accelerator* , instead of hc::accelerator& (so we don't have undefined semantic), or hc::accelerator (so we save memory).